x-pack/plugins/lists/server/services/exception_lists/exception_list_client_types.ts

```diff
@@ -113,18 +118,18 @@ export interface UpdateExceptionListItemOptions {
 export interface FindExceptionListItemOptions {
   listId: ListId;
   namespaceType: NamespaceType;
```

Comment:
Does the `savedObjectType` get appended to the `filter` server side? I see logic to determine `savedObjectType` in `find_exception_list_item.ts`, but in the sample script the `filter` string includes the `savedObjectType` already.

Reply:
It might be better that I don't require the saved object name to be pushed down but rather change it to use the name from the rest interface if it is asking for a filter against it being agnostic or not agnostic. I think I'm "leaking too much" in this case.

Reply:
I don't know, if I try to work around the "leakiness" it might cause a different set of issues. We should discuss this more I think as the UI is implemented and the agnostic switches are added.

Reply:
Trying to remember back to all the conversations around spaces. Right now does the list `type` determine the `namespaceType`?

Reply:
I have these TODO's I can solve real quick and provide an example. 

```ts
namespaceType: 'single', // TODO: Bubble this up
```

I think that would help clear this up so you know how you are going to query against a space agnostic list.

But basically you have two mapping space names depending on if you are trying to access a space agnostic list vs. a regular list. 

```
exception-list-agnostic
```

vs

```
exception-list
```


However, when you code up your filter where your UI needs are:
* I only want to look at the agnostic `exception list items`
* I want to filter against `entries.field` as the user types in the input box 
* I have the specific `exception list` GUID/name pre-determined that my UI knows about. This GUID/name is the pre-determined endpoint global `exception list` per deployment and let's use the example of this UUID: `028fd595-62ef-4824-ae01-7dc391366768` as the pre-determined `exception list` id of a global list.  

You would do this in REST:

```ts
POST /api/exception_lists/items/_find?namespace_type=agnostic&list_id=028fd595-62ef-4824-ae01-7dc391366768&filter=exception-list-agnostic.attributes.entries.field:actingProcess.file.signe*
``` 

where:

```
namespace_type=agnostic 
list_id=028fd595-62ef-4824-ae01-7dc391366768
filter=exception-list-agnostic.attributes.entries.field:actingProcess.file.signe*
```

Right now I haven't bubbled up `namespace_type` and tested the agnostic side of things. I should do that with this PR. I am also not allowing to query against _both_ the space agnostic and per space side of things just yet...If we want that then we would have _two_ different pre-determined list UUID's that are for per-space and space agnostic and the UI should query against each separate as that might make things easiest for us.

Reply:
Ok, I bubbled that code up and should be useable for exception_list and exception_list_items.

Reply:
Awesome! Thanks so much for the detailed reply. Seems to make sense to me.